spec/views/user/index.html.erb_spec.rb

```diff
@@ -4,11 +4,11 @@
   describe 'Index page' do
     before(:each) do
       @user1 = User.create(name: 'Gul',
-                           photo: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRJHYna2rR4LtBNlA3gUSGxsmaiYY4PyxQEcA&usqp=CAU', bio: 'Spring Sun', posts_counter: 0)
+                           photo: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRJHYna2rR4LtBNlA3gUSGxsmaiYY4PyxQEcA&usqp=CAU', bio: 'Spring Sun', email: 'example1@example', posts_counter: 0)
       @user2 = User.create(name: 'Ahmad',
-                           photo: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRJHYna2rR4LtBNlA3gUSGxsmaiYY4PyxQEcA&usqp=CAU', bio: 'Spring Sun', posts_counter: 0)
+                           photo: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRJHYna2rR4LtBNlA3gUSGxsmaiYY4PyxQEcA&usqp=CAU', bio: 'Spring Sun', email: 'example2@example', posts_counter: 0)
       @user3 = User.create(name: 'Muzafar',
-                           photo: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRJHYna2rR4LtBNlA3gUSGxsmaiYY4PyxQEcA&usqp=CAU', bio: 'Spring Sun', posts_counter: 0)
+                           photo: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRJHYna2rR4LtBNlA3gUSGxsmaiYY4PyxQEcA&usqp=CAU', bio: 'Spring Sun', email: 'example3@example', posts_counter: 0)
 
       visit users_path
     end
```

Comment:
### USE CAPYBARA TO WRITE INTEGRATION TESTS FOR EACH VIEW IN YOUR PROJECT

- [ ] Hey, you're meant to use capybara to write tests for the following `User index page`, `user show page`, 
`Post show page`, `User post index page`. Currently, you've not added any of these tests. kindly ensure you've added them. :pray: The test file should look something like `spec/features/post_index_spec.rb`

Reply:
Hi, hope to be fine all of these specs are in view/user and view/post files

Reply:
- [ ]  Great job implementing the integration test, but some of them still not passing, current m running 10 tests that fail,, please make sure to fix all failing tests 💯 👍🏻 Here is the screenshot 👇🏻



![image](https://user-images.githubusercontent.com/79058364/228430043-8294154a-57f1-4087-8a76-b09f762f01d0.png)